### ISSUE TITLE
Static helper doesn't use fingerprinting if no src/shared folder

### ISSUE THREAD

**Issue by barryf** [architect/architect#1148]
**Describe the issue**
If there is no `src/shared` folder, the `static()` helper function doesn't return the fingerprinted filename of assets.

**Steps to reproduce**
1. Set `fingerprint true` for `@static` in `app.arc`
2. Confirm your project has no `src/shared` folder
3. Wrap the URL to an asset with `arc.static(url)` in a route
4. Deploy the project and view the page
5. The asset URL is returned without fingerprint, e.g. `/_static/styles.css`

**Expected behavior**
A fingerprinted asset URL should be returned, e.g. `/_static/styles-fa34be6e1e.css`

**Additional context**
- Using Architect 8.5.11.
- Adding a `src/shared` folder and deploying then provides the fingerprinting as expected.
- [This line][^1] in the `static()` function checks for a `static.json` in `node_modules/@architect/shared` so I'm wondering if the deploy command is failing to copy it because no shared folder exists.

[^1]: https://github.com/architect/functions/blob/master/src/static/index.js#L17

**Comment by filmaj** [comment]
Agreed, just validated that this is not an issue in arc v11.2 either. Closing.
